src/ng-xform/form-control-layout/form-control-layout.component.html

```diff
@@ -9,7 +9,11 @@
 [formGroup]="fieldComponent.form">
   <label *ngIf="!fieldComponent.hideLabelOnEdit" for="{{fieldComponent.elementId}}" [ngClass]="labelStyleClass">
     {{ fieldComponent.field.label }}
-    <ng-xform-optional-tag [show]="fieldComponent.isOptional"></ng-xform-optional-tag>
+    <ng-xform-optional-tag *ngIf='!fieldComponent.isHorizontal' [show]="fieldComponent.isOptional"></ng-xform-optional-tag>
```

Comment:
One tag to show them all ;)

Can we implement the entire (optional) or asterisk logic in `<ng-xform-optional-tag>` component?